release.yaml

```diff
@@ -4,8 +4,12 @@ fleet-agent:
 - 100.1.0+up0.4.0-rc3
 fleet-crd:
 - 100.1.0+up0.4.0-rc3
+sriov:
```

Comment:
Can you fix the release.yaml indentations so that they are all consistent?

Reply:
It should be good now